torch/_subclasses/functional_tensor.py

```diff
@@ -278,6 +278,9 @@ def cuda(self, device=None, *args, **kwargs):
     int = _conversion_method_template(dtype=torch.int32)
     long = _conversion_method_template(dtype=torch.int64)
 
+    def to_dense(self):
+        return self.elem.to_dense()
```

Comment:
It's a 'torch._subclasses.functional_tensor.FunctionalTensor'. And yes, this solution does not feel right to me either, since having to add a relay for every method feels strange. But I am not sure what the better solution is either...

Reply:
For full type-background, given

```
    def to_dense(self):
        print('self', type(self))
        print('elem', type(self.elem))
        d = self.elem.to_dense()
        print('d', type(d))
        return d

```
then this is the output
```
self <class 'torch._subclasses.functional_tensor.FunctionalTensor'>
elem <class 'torch.Tensor'>
d <class 'torch._subclasses.functional_tensor.FunctionalTensor'>
```